### ISSUE TITLE
Style/SpaceInsideRangeLiteral should allow newlines

### ISSUE THREAD

**Issue by CyborgMaster** [rubocop/rubocop#1406]
Sometimes, when the parts of a range literal are long,  it is best to split them up over multiple lines.  While I agree that this is bad:

``` ruby
a ... b
```

I have the following in my code:

``` ruby
      @trigger.check([DateTime.parse('2014-05-12 04:00')...
                      DateTime.parse('2014-05-13 04:00')])
```

which RuboCop flags as incorrect.  Is there a better way to do what I need, or do we need to improve the cop?
